### PR TITLE
feat: add Justfiles

### PR DESCRIPTION
**Pull request by chrisgrieser**
`just` accepts `justfile` and `.justfile`, therefore adding both. https://just.systems/man/en/chapter_57.html?highlight=.justfile#hiding-justfiles

### CODE REVIEW

File: lua/nvim-web-devicons/icons-default.lua [hunk 1]
```diff
@@ -107,6 +107,12 @@ local icons_by_filename = {
     cterm_color = "28",
     name = "Gvimrc",
   },
+  [".justfile"] = {
+    icon = "",
```
Comment (alex-courtis) [inline lua/nvim-web-devicons/icons-default.lua:111]: Is that a just specific icon?

If not can we please use the general `"config"` icon ``

Comment (chrisgrieser) [reply lua/nvim-web-devicons/icons-default.lua:111]: I simply used the icon that `eza` is using for Justfiles. That airtight it should I change it?

Comment (alex-courtis) [reply lua/nvim-web-devicons/icons-default.lua:111]: Is it [eza](https://github.com/eza-community/eza) you are referring you to?

If so, how do we see their icon (source)?

Comment (chrisgrieser) [reply lua/nvim-web-devicons/icons-default.lua:111]: Yes. I think this is the location in their source code: https://github.com/eza-community/eza/blob/main/src/output/icons.rs#L281C4-L281C57

and in the terminal, shows like this: 
Pasted image 2024-06-07 at 02 01 04@2x

**Review by alex-courtis — APPROVED** [review]
Many thanks for your contribution!

**Comment by chrisgrieser** [comment]
> Wow... that looks very similar. We should probably sync up at some point.

That would indeed be nice for consistency between nvim and the terminal. Maybe establish a common "standard" for matching filetypes with nerdfont icons? eza and devicons cannot be the only ones who use those icons for filetypes. 

Anyway, that's a different topic, thanks for the merge.

**Pull request merged**